src/components/site-header/site-header.html.twig

```diff
@@ -49,6 +49,20 @@
               ],
             },
         },
+    - "custom_action" (associative array) (default: {}): optional custom link with optional popover, e.g.:
```

Comment:
can't we use the default ECL Link structure, instead of defining each fields?
I know that this is the case for the language switcher, but maybe we will be able to also update it in the future

Reply:
made it to use ecl link